controllers/policyset/policySetMapper.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package controllers
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+)
+
+func policySetMapper(c client.Client) handler.MapFunc {
```

Comment:
don't think this mapper is needed as it doesn't do anything special.

Reply:
On a second thought, I think this mapper is till needed. but update func should be updated to only enqueue when spec.policies is changed